test/ts/lazy.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+	createElement, Component,
+} from "../../src/";
+
+export interface LazyProps {
+        isProp: boolean;
+}
+// class IsLazy extends Component<LazyProps> {
+//         render ({ isProp }: LazyProps) {
+//                 return (
+//                         <div>{
+//                                 isProp ?
+//                                 'Super Lazy TRUE' :
+//                                 'Super Lazy FALSE'
+//                         }</div>
+//                 )
+//         }
+// }
+
+export const IsLazyComponent = (props: LazyProps) =>
```

Comment:
Could probably remove the commented out code? And potentially add a type annotation here, but that's not important

Reply:
I'm actually going to un-comment this section and have examples of both.